packages/coin-support-evm/src/operations/broadcastTransaction/index.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  getAccountAndCoin,
+  insertOrUpdateTransactions,
+} from '@cypherock/coin-support-utils';
+import { evmCoinList } from '@cypherock/coins';
+import { BigNumber } from '@cypherock/cysync-utils';
+import {
+  ITransaction,
+  TransactionStatusMap,
+  TransactionTypeMap,
+} from '@cypherock/db-interfaces';
+
+import { IBroadcastEvmTransactionParams } from './types';
+
+import { broadcastTransactionToBlockchain } from '../../services';
+
+export const broadcastTransaction = async (
+  params: IBroadcastEvmTransactionParams,
+): Promise<ITransaction> => {
+  const { db, signedTransaction, transaction } = params;
+  const { account, coin } = await getAccountAndCoin(
+    db,
+    evmCoinList,
+    transaction.accountId,
+  );
+  const isMine =
+    params.transaction.computedData.outputs[0].address ===
+    account.xpubOrAddress;
+
+  const txnHash = await broadcastTransactionToBlockchain(
+    signedTransaction,
+    coin.id,
+  );
+
+  const parsedTransaction: ITransaction = {
+    hash: txnHash,
+    fees: transaction.computedData.fee,
+    amount: '0',
+    status: TransactionStatusMap.pending,
+    type: TransactionTypeMap.send,
+    timestamp: Date.now(),
+    blockHeight: -1,
+    inputs: [],
+    outputs: [{ ...params.transaction.computedData.outputs[0], isMine }],
+    confirmations: 0,
+    accountId: account.__id,
+    walletId: account.walletId,
+    assetId: account.assetId,
+    familyId: account.familyId,
+  };
+
+  const amount = parsedTransaction.outputs.reduce(
+    (sum, output) => (output.isMine ? sum.plus(output.amount) : sum),
+    new BigNumber(0),
+  );
+  parsedTransaction.amount = amount.abs().toString();
```

Comment:
Logic looks incorrect. 

Current logic: If the output address is mine (Sending to myself), then it will be displayed in `amount`. 
Expected logic: If the output address is not mine (Sending to someone other than myself), then it will be displayed in `amount`.